### DISCUSSION TITLE
QUESTION: slim down live WebRTC playback?

### DISCUSSION THREAD

**Discussion by RotateAt60MPH** [OvenMediaLabs/OvenMediaEngine#1114] [Help] [answered]
We're trying to make the best out of inconsistent cell phone data bandwidth.  I'm trying to slim down the WebRTC playback of a live RTMP feed as much as possible to improve fps. 

Not ideal, I know, but any suggestions would be appreciated.  Change the resolution or bitrate?

Not sure how much this would help but is there a way to mute all audio from an OutputProfile in an Application?  

Thanks!

Here's our outputProfile currently:

```
{
    "message": "OK",
    "response": {
        "encodes": {
            "audios": [
                {
                    "bypass": true,
                    "name": "bypass_audio"
                },
                {
                    "bitrate": "128000",
                    "channel": 2,
                    "codec": "opus",
                    "samplerate": 48000
                }
            ],
            "videos": [
                {
                    "bypass": true,
                    "name": "bypass_video"
                }
            ]
        },
        "name": "bypass_stream",
        "outputStreamName": "${OriginStreamName}"
    },
    "statusCode": 200
}
```

**Comment by getroot** [comment, marked as answer]
I think ABR is probably what you want. 
https://airensoft.gitbook.io/ovenmediaengine/streaming/webrtc-publishing#adaptive-bitrates-streaming-abr
